messageService/message.go

```diff
@@ -13,6 +14,16 @@ type Authentification struct {
 	NickName string
 }
 
+//Message for online contacts after login
+type Contact struct {
```

Comment:
We'll have Contact struct after merging PR#38. Maybe, rename this struct to ContactsOnline?